### PR TITLE
Added an empty registration page

### PR DESCRIPTION
**Pull request by foadshariat7**

### CODE REVIEW

File: client-app/src/App.js [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 import LoginPage from './LoginPage';
+import RegistrationPage from './RegistrationPage';
 
 const App = () => {
     return (
         <Router>
             <Routes>
                 <Route path="/" element={<LoginPage />} />
-                {/* Add more routes for other pages here */}
+                <Route path="/register" element={<RegistrationPage />} /> {/* Add this line */}
```
Comment (chairnetmuche) [inline client-app/src/App.js:12]: I think it looks good to me. Good Job Foad!!

File: client-app/src/LoginPage.js [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
+import { useNavigate } from 'react-router-dom';
 
 const LoginPage = () => {
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
+    const navigate = useNavigate();
 
     const handleSubmit = (event) => {
         event.preventDefault();
         console.log('Email:', email, 'Password:', password);
```
Comment (chairnetmuche) [inline client-app/src/LoginPage.js:14]: Should we remove this line....I don't think it's necessary to have it

**Review by chairnetmuche — APPROVED** [review]
Please take a look at it when you have time. Thank you!

**Review by foadshariat7 — COMMENTED** [review]
The changes are reviewed

**Pull request merged**